docs/modules/user-guide/partials/proc_migrating-schema-version.adoc

```diff
@@ -6,7 +6,7 @@ This section describes how to migrate the existing schema version to a v2.x devf
 
 .Procedure
 
-. To migrate a schema version from a v1.x devfile to a v2.x devfile, replace `apiVersion: 1.0.0` with `schemaVersion: 2.0.0`:
+. To migrate a schema version from a v1.x devfile to a v2.x devfile, replace `apiVersion: 1.0.0` with `schemaVersion: 2.1.0`:
```

Comment:
Should we include information on migrating from v2.0 to v2.1 as well?

Reply:
That's a good catch I'm not sure about.
I expect that 2.1 comparing to 2.0 is an extension.
Users can just update schemaVersion to 2.1.0 and use newer format features.